examples/graphql-react-apollo/src/ApolloClient.js

```diff
@@ -1,11 +1,18 @@
-import ApolloClient from 'apollo-boost'
+import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client'
 
-// Isolate Apollo client so it could be reused
-// in both application runtime and tests.
-export const client = new ApolloClient({
+const cache = new InMemoryCache()
+
+const link = new HttpLink({
   uri: 'http://localhost:3000/graphql',
 
   // Use explicit `window.fetch` so tha outgoing requests
   // are captured and deferred until the Service Worker is ready.
   fetch: (...args) => fetch(...args),
 })
+
+// Isolate Apollo client so it could be reused
+// in both application runtime and tests.
+export const client = new ApolloClient({
+  cache,
```

Comment:
`cache` is a required property, so a I created a simple `InMemoryCache` and passed it in